container.css

```diff
@@ -0,0 +1,40 @@
+.subcontainer{
```

Comment:
naming convention in css has to be corrected, subcontainer needs to be sub-container if its two words